packages/dds/merge-tree/src/test/revertibles.spec.ts

```diff
@@ -181,9 +181,9 @@ describe("MergeTree.Revertibles", () => {
 		logger.validate({ baseText: "123" });
 	});
 
-	it("revert overlapping remove", () => {
+	it("revert overlapping remove of multiple segments", () => {
```

Comment:
if we want to make this a separate test case that's ok with me. I don't want to do too much refactoring here though--i'd probably just dupe the bulk of the code for now (maybe we can better abstract it later)

Reply:
i would like to keep both. can you just wrap it in a loop of the different inputs